### ISSUE TITLE
beta example seems broken?

### ISSUE THREAD

**Issue by cjayb** [jonescompneurolab/hnn-core#407]
What happened to the ERP-modulation due to a beta-event, @ntolley ?

https://jonescompneurolab.github.io/hnn-core/dev/auto_examples/workflows/plot_simulate_beta.html#sphx-glr-auto-examples-workflows-plot-simulate-beta-py

![sphx_glr_plot_simulate_beta_003](https://user-images.githubusercontent.com/5610076/127648246-9a99bd51-742d-4a16-bc73-54ac40537b58.png)

**Comment by ntolley** [comment]
Checkout the discussion after #383 was merged. The new seeding schema no longer produces coincident spike times that activate the L2 basket cells. While it's not ideal to have such an example that is so sensitive to seeding, it's how the results were achieved in the publication.

@rythorpe @jasmainak did you guys settle on a path to resolving this? The other options are to hard code the spike times (really not ideal), or change the input weights. Of course changing the weights would deviate from publication values so there may need to be added discussion about why it's necessary.

**Comment by jasmainak** [comment]
yeah, this really bothers me that examples are so sensitive to seeding. This one also broke:

https://jonescompneurolab.github.io/hnn-core/dev/auto_examples/howto/plot_simulate_mpi_backend.html#sphx-glr-auto-examples-howto-plot-simulate-mpi-backend-py

although probably not by as much

**Comment by cjayb** [comment]
> Checkout the discussion after #383 was merged.

My bad. Feel free to close this, but I'm sure we're all agreed on @jasmainak 's sentiment about this...

**Comment by rythorpe** [comment]
The general issue still needs resolution, but I'm closing this in favor of its twin issue, #400.
